src/query/service/src/interpreters/access/privilege_access.rs

```diff
@@ -182,13 +182,25 @@ impl PrivilegeAccess {
         db_name: &str,
         table_name: &str,
         privileges: Vec<UserPrivilegeType>,
+        if_exists: bool,
```

Comment:
`if_exists` means only validate the table access when the `catalog`, `db`, `table` exists, right?

Reply:
> `if_exists` means only validate the table access when the `catalog`, `db`, `table` exists, right?

When the catalog exists.